huaweicloud/services/acceptance/modelarts/resource_huaweicloud_modelarts_devserver_test.go

```diff
@@ -88,7 +106,7 @@ func TestAccDevServer_basic(t *testing.T) {
 	})
 }
 
-func testDevServer_basic(name, password string, autoRenew bool) string {
+func testDevServer_basic_step1(name, password string, autoRenew bool) string {
```

Comment:
rename it as the `testAccDevServer_basic` and using these references as follows:
```
// Rename the argument input 'autoRenew' as the 'isAutoRenew', and put it ahead of the other string arguments.
func testAccDevServer_basic(isAutoRenew bool, name, password string) string {
  return testAccDevServer_base(isAutoRenew, name, password)
}
```